Display CTR data inline (#214)
mozilla/skylight: CHANGELOG.md

## Friday, June 7th, 2024

### Added

> * When enabling Looker, CTR percents will be displayed in the Metrics column inside tables

Author: ```suggestion
* Display CTR percentages in the table when Looker is configured
```
